imap_processing/codice/codice_l1a_hi_omni.py

```diff
@@ -47,7 +47,7 @@ def l1a_hi_omni(unpacked_dataset: xr.Dataset, lut_file: Path) -> xr.Dataset:
     plan_step = unpacked_dataset["plan_step"].values[0]
 
     logger.info(
-        f"Processing species with - APID: {apid}, View ID: {view_id}, "
+        f"Processing species with - APID: 0x{apid:X}, View ID: {view_id}, "
```

Comment:
Maybe put both in the output? I think most instruments are using the integer value.
```suggestion
        f"Processing species with - APID: {apid} / 0x{apid:X}, View ID: {view_id}, "
```

Reply:
That I can. I can see someone using the log for looking up in LUT which stores hex only.